selfdrive/ui/ui.h

```diff
@@ -107,50 +102,52 @@ typedef struct UIScene {
   uint64_t started_frame;
 } UIScene;
 
-typedef struct UIState {
-  VisionIpcClient * vipc_client;
-  VisionIpcClient * vipc_client_front;
-  VisionIpcClient * vipc_client_rear;
-  VisionBuf * last_frame;
+class UIState {
```

Comment:
what's the benefit of making this a class? let's just do the minimal refactor to remove the other readers

Reply:
the benefit is that we can use the syntax of ` s->sm["radarState"] `to get the reader, otherwise we can only use `(*(s->sm))["radarState"] `. (or use A temporary variable, such as `Submaster &sm = *(s->sm)`)